src/ApiService/ApiService/Functions/Jobs.cs

```diff
@@ -71,7 +71,16 @@ public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
         var r = await _context.JobOperations.Insert(job);
         if (!r.IsOk) {
             _logTracer.WithTag("HttpRequest", "POST").WithHttpStatus(r.ErrorV).Error($"failed to insert job {job.JobId:Tag:JobId}");
+            return await _context.RequestHandling.NotOk(
+                req,
+                new Error(
```

Comment:
@Porges  you mean like that ?
@chkeita  - yes it fails

Reply:
@stishkin I think probably just `UNABLE_TO_CREATE`; there is no container setup here?

Reply:
@chkeita  - what do you mean ? job created should be sent when it failed to insert ?

Reply:
oh never mind i misunderstood the call